test/IntegrationTests/Helpers/EnvironmentHelper.cs

```diff
@@ -223,6 +223,12 @@ public static bool IsRunningOnCI()
             environmentVariables["OTEL_EXPORTER_OTLP_ENDPOINT"] = $"http://localhost:{testSettings.MetricsSettings.Port}";
         }
 
+        if (testSettings.LogSettings != null)
+        {
+            environmentVariables["OTEL_LOGS_EXPORTER"] = testSettings.LogSettings.Exporter;
```

Comment:
You should extend `ClearProfilerEnvironmentVariables` for these variables (the same class).